docs/running_psalm/issues/PossiblyUnusedMethod.md

```diff
@@ -9,6 +9,7 @@ with `@psalm-api`.
 ```php
 <?php
 
+/** @psalm-api */
```

Comment:
This is supposed to be a snippet that emits `PossiblyUnusedMethod`. I think we have a test that makes sure the issue being documented is emitted by the example the doc for that issue contains.

Reply:
Yep noticed that, will have to either revert or patch the test